src/Microsoft.AspNet.Authentication/AuthenticationOptions.cs

```diff
@@ -27,6 +27,13 @@ public string AuthenticationScheme
         }
 
         /// <summary>
+        /// If true the authentication middleware alter the request user coming in and
```

Comment:
Filed https://github.com/aspnet/Security/issues/190 to track this general issue, as I'm pretty bad at updating docs in general, I'm sure they are pretty out of date now